lesson05/script.js

```diff
@@ -0,0 +1,25 @@
+const ul = document.getElementById('js-list');
+const lists = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+   ]
+
+const promise = new Promise(resolve => {
+    resolve(lists)
+});
+
+promise.then(() => {
+    const fragment = document.createDocumentFragment();
+    for(const list of lists){
```

Comment:
Must

I overlooked an important place. I'm sorry.

If it's a `const list of lists `, I'm going to refer to `const lists`instead of receiving the resolved value. What should I do?

Reply:
Thank you for your comment!

I was very misunderstood.
I had to store the value received by the resolve in a variable.